docs/AccountIntegration.md

```diff
@@ -5,10 +5,10 @@
 | Name | Type | Description | Notes |
 | ---- | ---- | ----------- | ----- |
 | **name** | **String** | Company name. |  |
-| **categories** | **Array&lt;String&gt;** | Category or categories this integration belongs to. | [optional] |
-| **image** | **String** | Company logo in rectangular shape. | [optional] |
-| **square_image** | **String** | Company logo in square shape. | [optional] |
-| **color** | **String** | The color of this integration used for buttons and text throughout the app and landing pages. | [optional] |
+| **categories** | [**Array&lt;CategoriesEnum&gt;**](CategoriesEnum.md) | Category or categories this integration belongs to. Multiple categories should be comma separated.&lt;br/&gt;&lt;br&gt;Example: For [ats, hris], enter &lt;i&gt;ats,hris&lt;/i&gt; | [optional] |
+| **image** | **String** | Company logo in rectangular shape. &lt;b&gt;Upload an image with a clear background.&lt;/b&gt; | [optional] |
+| **square_image** | **String** | Company logo in square shape. &lt;b&gt;Upload an image with a white background.&lt;/b&gt; | [optional] |
+| **color** | **String** | The color of this integration used for buttons and text throughout the app and landing pages. &lt;b&gt;Choose a darker, saturated color.&lt;/b&gt; | [optional] |
```

Comment:
nit: these comments are internal (for adding stuff in the CMS). I've been pretty inconsistent about whether I remove these from the sdks, I'll leave it up to you whether its work the effort.